lib/rspec/rails/configuration.rb

```diff
@@ -69,7 +69,12 @@ def self.initialize_configuration(config) # rubocop:disable Metrics/MethodLength
       config.add_setting :use_transactional_fixtures, alias_with: :use_transactional_examples
       config.add_setting :use_instantiated_fixtures
       config.add_setting :global_fixtures
+
+      config.add_setting :fixture_paths
+
+      # @deprecated TestFixtures#fixture_path is deprecated and will be removed in Rails 7.2
       config.add_setting :fixture_path
+
```

Comment:
We shouldn't add the new setting on Rails that doesn't support it, and the implementation of this original method should take into account the existence of the new one:
```suggestion
      if ::Rails::VERSION::STRING < '7.1.0'
        config.add_setting :fixture_paths
        # psuedo coding here, this should probably be in the instance_exec below
        def config.fixture_path=(path)
          RSpec.deprecate(
            "config.fixture_path =  #{path.inspect}",
            replacement: "config.fixture_paths = [#{path.inspect}],
            message: "Rails 7.1 has deprecated the singular fixture_path in favour of an array"
          )

          self.fixture_paths = [path]
        end
      else
        config.add_setting :fixture_path
      end
```
The message is optional for deprecation, and could be reworded

Reply:
Fixed